photo.py

```diff
@@ -0,0 +1,9 @@
+class Photo:
+    def __init__(self, id):
```

Comment:
Accept all attributes as arguments, so you don’t have to set them via .attr after object creation

Reply:
done